modules/ROOT/pages/find-api-id-task.adoc

```diff
@@ -5,14 +5,8 @@ endif::[]
 
 API Manager generates the `apiId` of new APIs managed by API Manager for use with Mule 4. The API ID is displayed on the API instance dashboard if you select Mule 4 Runtime in the configuration of an API instance.
 
-. On the API administration dashboard, expand the API version name to display the instances.
+. On the API administration dashboard, click an API name.
 +
-image::apis-api-manager-dashboard.png[]
+The API ID is displayed under *API Instance*. The API ID is not available for Mule 3 use in API Manager.
```

Comment:
```suggestion
The API ID is displayed under *API Instance*. The API ID is not available for Mule 3 use in API Manager.
```

Reply:
@chris-campbell9 what do you mean with this line?
API ID IS available for both Mule 3 & 4.